tests/test_utils.py

```diff
@@ -183,6 +183,7 @@ def test_get_structured_config_data(test_cls_or_obj: Any, expectation: Any) -> N
         assert d["dict1"] == {}
 
 
+@pytest.fixture  # type: ignore
```

Comment:
what's with the @pytest.fixtures?

Reply:
I mentioned it above. When I switched to python 3.7 running pytest raised and error that said that a fixture was missing.

Reply:
How come it was not an issue before this diff?

Reply:
I've been on python 3.6 the whole time.

Reply:
Were you on a greater version ?

Reply:
1. I am on 3.8
2. nox is testing 3.6, 3.7 and 3.8 in CI.

Reply:
Wait now it doesn't raise the error. My bad i don't know how did it happen

Reply:
We said we will remove this fixture.

Can you remove?
also, can you go over the whole diff to be sure it all makes sense? I feel like I keep finding resurrected things.

Reply:
You missed my request again.